src/types/Patterns.test.ts

```diff
@@ -68,6 +69,11 @@ describe('Input Pattern Tests', () => {
       isLegalEntity(expr)
     )
   })
+  it('validates Region pattern', () => {
+    validate(REGION_TEST_DATA.REGION, (expr: string) =>
+      isRegion(expr)
+    )
```

Comment:
I think you don't need that anonymous arrow function wrapper and can simply write

`validate(REGION_TEST_DATA.REGION, isRegion)`